numismatic/feeds/cryptocompare.py

```diff
@@ -9,13 +9,24 @@
 import attr
 
 from .base import Feed, RestClient
+from ..events import PriceUpdate, Ticker
 from ..libs.utils import date_range, make_list_str, to_datetime, \
     dates_and_frequencies
 
 
 logger = logging.getLogger(__name__)
 
 
+EXCHANGES = ('BTC38,BTCC,BTCE,BTER,Bit2C,Bitfinex,Bitstamp,Bittrex,CCEDK,'
```

Comment:
WOW :+1: 

However, rather than have things embedded here, what do you think about moving it to the `numismatic.ini` file. For example (taking a random exchange), using `Korbit`:

```
[Korbit]
CryptoCompareExchange = Korbit
```

Reply:
I agree